server/manifestmergetool/src/test/java/com/defold/manifestmergetool/ManifestMergeToolTest.java

```diff
@@ -878,9 +878,9 @@ public void testMergeHTML5() throws IOException {
             + "<!doctype html>\n"
             + "<html>\n"
             + " <head></head>\n"
-            + " <body> \n"
```

Comment:
After updating to jsoup 1.15+ it's vital to remove spaces in the end of string to pass the test.